src/app/containers/ScheduleControls.tsx

```diff
@@ -4,84 +4,172 @@ import { connect } from 'react-redux';
 import ScheduleShiftAddModal from './ScheduleShiftAddModal';
 import ScheduleShiftCopyModal from './ScheduleShiftCopyModal';
+import { IdType } from '../api/Api';
 
 interface ScheduleControlsStateProps {
     visibleTimeStart: any;
     visibleTimeEnd: any;
 }
 
 interface ScheduleControlsProps {
+    submit?: () => void;
+    cancel?: () => void;
+    deleteShift?: (shiftIds: IdType[]) => void;
+    selectedShifts?: IdType[];
 }
 
 interface ScheduleDistpatchProps {
     updateVisibleTime: (startTime: any, endTime: any) => void;
+    showShiftCopyModal: () => void;
+    showShiftAddModal: () => void;
 }
 
 class ScheduleControls extends React.PureComponent<
     ScheduleControlsProps & ScheduleControlsStateProps & ScheduleDistpatchProps> {
 
     render() {
-        const { visibleTimeStart, visibleTimeEnd, updateVisibleTime } = this.props;
+        const {
+            visibleTimeStart,
+            visibleTimeEnd,
+            updateVisibleTime,
+            showShiftCopyModal,
+            showShiftAddModal,
+            submit,
+            cancel,
+            // tslint:disable:no-shadowed-variable
+            deleteShift,
+            selectedShifts = []
+        } = this.props;
+
         return (
-            
-            <div style={{ textAlign: 'center', display: 'flex'}}>
-                <Button
-                    onClick={() => updateVisibleTime(
-                        moment(visibleTimeStart).subtract('week', 1),
-                        moment(visibleTimeEnd).subtract('week', 1)
-                    )}
-                    bsStyle="link" 
-                    bsSize="large" 
-                    style={{color: 'white'}}
+            <div
+                style={{
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    width: '100%',
+                    paddingLeft: 200
+                }}
+            >
+                <div
+                    style={{
+                        margin: 10,
+                        paddingRight: 15
+                    }}
                 >
-                    <Glyphicon glyph="chevron-left" />
-                </Button>
-                
-                <CalendarButton 
-                    onChange={(selectedDate) => updateVisibleTime(
-                        moment(selectedDate).startOf('week').add(1, 'day'),
-                        moment(selectedDate).endOf('week').subtract(1, 'day')
-                    )}
-                    defaultValue={visibleTimeStart}
-                    todayOnClick={() => updateVisibleTime(
-                        moment().startOf('week').add(1, 'day'),
-                        moment().endOf('week').subtract(1, 'day')
-                    )}
-                />
+                    <ScheduleShiftMultiEditForm
+                        onApply={
+                            () => {
+                                submit && submit();
+                                cancel && cancel();
```

Comment:
perhaps reset / clear would be more appropriate for this method name?  (I think you said that you changed the name of this button in the UI to be clear instead of cancel)